test/components/polylith/bricks/test_base.py

```diff
@@ -0,0 +1,37 @@
+from test import params
+
+import pytest
+
+from components.polylith.bricks.base import create_base, get_bases_data
+
+
+@pytest.mark.parametrize(
+    "id, expected_dirs, expected_dir_structure",
+    params.create_base_params,
+    ids=params.create_base_ids,
+)
+def test_create_base(handle_workspace_files, id, expected_dirs, expected_dir_structure):
+
+    create_base(
+        path=handle_workspace_files,
+        namespace="test_namespace",
+        package="test_package",
+        description="test desc",
+    )
+    results = [
+        x for x in handle_workspace_files.iterdir() if x.name != "workspace.toml"
+    ]
+
+    assert handle_workspace_files.is_dir()
```

Comment:
Is this assertion needed? If I understand the fixture, it is created in there and not by the code under test. Maybe I'm mistaking things 😄

Reply:
Good catch! I missed that clean up.